apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/registry/configuration/ApolloServiceRegistryAutoConfigurationNotEnabledTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.biz.registry.configuration;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+
+@SpringBootTest
+@ContextConfiguration(classes = {
+    ApolloServiceRegistryAutoConfiguration.class,
+    ApolloServiceDiscoveryAutoConfiguration.class
+})
+class ApolloServiceRegistryAutoConfigurationNotEnabledTest {
```

Comment:
What does this test case test?

Reply:
Ensure that this pr won't cause configservice or adminservice startup fail when database discovery doesn't enabled.

Reply:
Good point! Then I would suggest we also add some verifications, e.g. 

```java
  @Autowired
  private ApplicationContext context;

  @Test
  void load() {
    Assertions.assertTrue(context.getBeansOfType(ServiceRegistryService.class).isEmpty());
    // and more ...
  }
```

Reply:
I have add those bean check in newest commit